pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotTableRestletResource.java

```diff
@@ -171,22 +171,14 @@ private String listTableConfigs(@Nonnull String tableName, @Nullable String tabl
       "Get/Enable/Disable/Drop a table. If table name is the only parameter specified "
           + ", the tableconfig will be printed")
   public String alterTableStateOrListTableConfig(
-      @ApiParam(value = "Name of the table", required = false) @PathParam("tableName") String tableName,
-      @ApiParam(value = "enable|disable|drop", required = false) @QueryParam("state") String stateStr,
-      @ApiParam(value = "realtime|offline", required = false) @QueryParam("type") String tableTypeStr) {
+      @ApiParam(value = "Name of the table", required = true) @PathParam("tableName") String tableName,
```

Comment:
This is a change to the API. Are we sure that no one uses this to list tables? Does another API provide similar functionality?

Reply:
It's a path parameter, which means it's always required